checks/tasks/__init__.py

```diff
@@ -38,10 +38,10 @@ def ub_ctx(self):
                 self._ub_ctx.zone_add("test.", "transparent")
 
             self._ub_ctx.set_option("cache-max-ttl:", str(settings.CACHE_TTL * 0.9))
+            self._ub_ctx.set_option("cache-max-negative-ttl:", str(settings.CACHE_TTL * 0.9))
             # Some (unknown) tests probably depend on consistent ordering in unbound responses
             # https://github.com/internetstandards/Internet.nl/pull/613#discussion_r892196819
             self._ub_ctx.set_option("rrset-roundrobin:", "no")
```

Comment:
This change normally would have no effect. `cache-max-ttl` applies to all records in the cache. `cache-max-negative-ttl` applies after that only for negative records (NXDOMAIN, NOERROR) as an exception for those records. Since `cache-max-ttl` would normally be a value of less than 1 hour (default `cache-max-negative-ttl`) for internet.nl the change is moot. However, explicitly setting it as the same value would guarantee homogeneous  behavior across different setups and configurations.

Reply:
Hmm ok, I will apply it then - original reason was a report that the test was not picking up new DNS records, then I looked in unbound config to find no cache-max-ttl at all, then realised it would be in the unbound context. So the original problem is likely not related to this, but it's still a good consistency fix.

Reply:
If caching was a problem for new DNS records (especially in DNSSEC signed zones) then #495 may still be related. I changed its title because it was not only DNSSEC related.
Also in batch mode, all the pyunbounds forward to a central Unbound IIRC. That one also needs the proper TTL max values in its configuration.